amplivelist/amp-live-list.go

```diff
@@ -50,7 +50,17 @@ type Page struct {
 
 var blogs []BlogItem
 
-func InitBlogs() {
+func Init() {
```

Comment:
Let's move the string concatenation into registerHandler as well. This helps further reduce duplication (namely how the paths are build).